composer.json

```diff
@@ -25,22 +25,17 @@
     "issues": "https://github.com/postcode-nl/PostcodeNl_Api_Client/issues",
     "support": "https://github.com/postcode-nl/PostcodeNl_Api_Client"
   },
-  "repositories": [
-    {
-      "type": "vcs",
-      "url": "https://github.com/postcode-nl/PostcodeNl_Api_Client"
-    }
-  ],
   "autoload": {
-    "psr-0": {
-      "PostcodeNl\\Api": "src/"
-    },
-    "classmap": [
-      "src/PostcodeNl/Api/Client.php"
-    ]
+    "psr-4": {
+      "PostcodeNl\\Api\\": "src"
```

Comment:
Add a trailing slash to "src" i.e. "src/" is preferred.

Reply:
This is just a matter of preference it isn't required, I'll change it.